eth/src/main/kotlin/com/d3/eth/sidechain/EthChainListener.kt

```diff
@@ -95,7 +98,7 @@ class EthChainListener(
      */
     private fun publishEthBlockAndSaveHeight(ethBlock: EthBlock) {
         ethBlocksSubject.onNext(ethBlock)
-        val height = ethBlock.block.number
+        val height = ethBlock.block.number.add(bigIntOne)
```

Comment:
```suggestion
        val height = ethBlock.block.number.inc()
```

Reply:
tochnyak

Reply:
But it will lead to creation of BigInt object every time btw